mac

```diff
@@ -146,9 +149,9 @@ install_asdf() {
     brew install asdf
 
     # shellcheck disable=SC2016
-    append_to_zshrc 'export PATH=~/.asdf/shims:$PATH"'
+    append_to_zshrc 'export PATH="$HOME/.asdf/shims:$PATH"'
 
-    append_to_zshrc "source $(brew --prefix asdf)/asdf.sh" 1
+    append_to_zshrc "source $(brew --prefix asdf)/libexec/asdf.sh" 1
```

Comment:
I found a thread in [a discussion](https://github.com/asdf-vm/asdf/issues/891#issuecomment-799423553) that seems to suggest that this is not the correct fix to make:

![image](https://user-images.githubusercontent.com/696529/220563058-9784cd00-afae-4a24-8ab9-5938c9fb7646.png)

The root cause seems to how `PATH`  is defined. So I am wondering if the change from `~/` to `$HOME` is not causing issues.

Reply:
Thanks for the additional resources, specially the one for [formulae.brew.sh/formula/asdf](https://formulae.brew.sh/formula/asdf). I agree to add the changes for `asdf`.